### DISCUSSION TITLE
If I have LND set up on my device, will it be enough to start using RGB for issuing assets, transferring them and accepting?

### DISCUSSION THREAD

**Discussion by UkolovaOlga** [LNP-BP/presentations#40] [Q&A]
### TL;DR
- RGB asset issuance does not happen on the level of Lightning, it happens on Bitcoin L1.
- the only node that currently supports RGB is the [LNP Node](https://github.com/LNP-BP/lnp-node).

### TS;WM

To start with, you can’t issue asset on the LN, it does not make a lot of sense as due to the Lightning Network architecture, this asset would be too ephemeral. The only place where you can issue an asset is onchain bitcoin transaction, similar to the only place where you can create a channel being onchain transaction/funding transaction. So, RGB assets **issuance** is in no way related to the LN. 
For asset issuance we already have a toolkit that is named [Bitcoin Pro](https://github.com/pandoracore/bitcoin-pro) and was developed by [Pandora Core](https://pandoracore.com/) company. This tool allows you to create and issue assets of different forms (currently it’s fungible, but the plan is to add support of the non-fungible assets soon).  

The only LN node that is currently working with RGB is the LNP Node and the only way you can use these assets in the LN is to install LNP Node and open Lightning channels with it. Today, once you have an asset issued, you can open channels only between LNP Nodes. In the next few months we will add interoperability with other LN nodes, so you will be able to open bitcoin channels with those nodes too. However, currently if you'd be adding RGB assets to the channel while the second node is not the LNP Node, it will simply not understand or see any assets as it does not support RGB. We are working with many teams though and, hopefully, RGB support will be added to other nodes, including LND, over the course of the next months. More precise timing heavily depends on the teams that develop corresponding nodes.

 To emphasize again, RGB itself if 100% LN compliant, meaning that it does not require any changes to the LN itself. It requires some specific changes in concrete lightning nodes implementations adding more functionality to them, but big part of this is out of our control and totally up to the corresponding dev teams.
